<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-SLURSDT2" failures="0" tests="1" name="com.webtest.dataprovider.NSDataProvider" time="1.018" errors="1" timestamp="15 10月 2018 13:30:03 GMT" skipped="0">
  <testcase classname="com.webtest.dataprovider.NSDataProvider" name="test1" time="1.018">
    <error message="
Data provider mismatch
Method: test1([Parameter{index=0, type=org.testng.ITestContext, declaredAnnotations=[]}])
Arguments: [(java.lang.String) 蛋仔,(java.lang.String) 蛋仔,(java.lang.String) 蛋仔,(java.lang.String) 蛋仔,(java.lang.String) 蛋国,(java.lang.String) 蛋国国语,(java.lang.String) 42772,(java.lang.String) 90,(java.lang.String) D:\\twenty.jpg,(java.lang.String) 蛋蛋是个好孩子，还是个帅小孩]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
Data provider mismatch
Method: test1([Parameter{index=0, type=org.testng.ITestContext, declaredAnnotations=[]}])
Arguments: [(java.lang.String) 蛋仔,(java.lang.String) 蛋仔,(java.lang.String) 蛋仔,(java.lang.String) 蛋仔,(java.lang.String) 蛋国,(java.lang.String) 蛋国国语,(java.lang.String) 42772,(java.lang.String) 90,(java.lang.String) D:\\twenty.jpg,(java.lang.String) 蛋蛋是个好孩子，还是个帅小孩]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:45)
at org.testng.internal.Parameters.injectParameters(Parameters.java:796)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:982)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- test1 -->
</testsuite> <!-- com.webtest.dataprovider.NSDataProvider -->
